src/zcl_abapgit_repo.clas.abap

```diff
@@ -222,15 +222,19 @@ CLASS ZCL_ABAPGIT_REPO IMPLEMENTATION.
 
   METHOD deserialize_checks.
 
-    DATA: lt_requirements TYPE zif_abapgit_dot_abapgit=>ty_requirement_tt.
+    DATA: lt_requirements    TYPE zif_abapgit_dot_abapgit=>ty_requirement_tt,
+          lv_master_language TYPE spras.
 
 
     find_remote_dot_abapgit( ).
 
+    lv_master_language = get_dot_abapgit( )->get_master_language( ).
+
     IF get_local_settings( )-write_protected = abap_true.
       zcx_abapgit_exception=>raise( 'Cannot deserialize. Local code is write-protected by repo config' ).
-    ELSEIF get_dot_abapgit( )->get_master_language( ) <> sy-langu.
-      zcx_abapgit_exception=>raise( 'Current login language does not match master language' ).
+    ELSEIF lv_master_language <> sy-langu.
+      zcx_abapgit_exception=>raise( |Current login language '{ sy-langu }' |
+                                 && | does not match master language '{ lv_master_language }'| ).
```

Comment:
I think the string template prevents the conversion exit for spras and shows the 1 character long language code (looking at the screenshot you posted earlier). The two character ISO one might be easier to read. (Also sy-langu again)

Reply:
Fixed
![grafik](https://user-images.githubusercontent.com/17437789/48856471-c03f3a00-edb6-11e8-90fe-542d41541140.png)